doc/ping.xml

```diff
@@ -357,7 +357,8 @@ xml:id="man.ping">
           <emphasis remap="I">mark</emphasis> to tag the packets
           going out. This is useful for variety of reasons within
           the kernel such as using policy routing to select
-          specific outbound processing.</para>
+          specific outbound processing.
+          CAP_NET_ADMIN capability is required.</para>
```

Comment:
You could also refer to `man 7 socket`:

```
        SO_MARK (since Linux 2.6.25)
               Set  the  mark  for  each  packet sent through this socket (similar to the netfilter MARK target but socket-based).  Changing the mark can be used for mark-based routing without netfilter or for packet filtering.
               Setting this option requires the CAP_NET_ADMIN capability.
```
as an authoritative source for this, here, or in another spot where this is suitable.